src/components/WebexMeeting/WebexMeeting.scss

```diff
@@ -27,7 +27,7 @@
   }
 
   .loading-logo {
-    background: var(--wxc-loading-logo) no-repeat center / 6.5rem 4.627rem;
+    background: var(--wxc-loading-logo) no-repeat center / 11.6875rem;
```

Comment:
Can you make this a variable or formulate as to how you arrived to this number?

Reply:
Hi @lalli-flores,

I received in Jira, from Hannah, the correct dimensions for loading the logo, which should be 187 * 187 px. I transformed these pixels into rem 187/16 = 11.6875 and used the size in rem. That's it. I will add a new variable for that dimension and add it in both themes. Thank you.

Reply:
Thanks for the clarification @lungancatalin. Someone without access or knowledge of the Figma spec would look at the number and wonder where this seemingly random number came from.